types/address.go

```diff
@@ -35,8 +31,13 @@ const (
 	//	config.SetCoinType(yourCoinType)
 	//	config.Seal()
 
+	// TODO HV2: all prefixes are now empty strings.
+	//  Another solution (closer to what's being done in cosmos-sdk) would be to use "0x" as our prefix.
```

Comment:
I think this answers some of the comments I added previously.
If you go for the "0x as prefix" approach, I would be a good idea to revert the changes made in this PR that change `hex.DecodeString` to `common.FromHex` (e.g., function `GetFromHex`in `types/address.go`)

Reply:
Correct, `common.FromHex` is underlying calling `hex.DecodeString`, contemplating the existence of a `0x` prefix. Will check on feasibility of using `0x` or completely remove the prefix (the latter approach is used in current `heimdall`).

Reply:
Irrelevant now as the prefix has been removed

Reply:
Resolved